### PR TITLE
LRDOCS-4745 and search deployment section reorganization

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-4745 installing solr section, and section reorganization

### CODE REVIEW

File: discover/deployment/articles/01-installing-search-engine/03-installing-solr/00-installing-solr-intro.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+[Elasticsearch](/discover/deployment/-/knowledge_base/7-1/configuring-elasticsearch-for-liferay-0)
+is the default search engine that ships with @product@, it's perfectly valid to
+use Solr instead. In particular, if you've already been using Solr with a
+previous version of @product@, or your deployment system (for example, your OS or
+JVM) [isn't supported by Elasticsearch](https://www.elastic.co/support/matrix),
+you might choose to use Solr to search and index your @product@ data.
+
+There are circumstances that force you to use Elasticsearch instead of Solr.
+Read
+[here](/discover/deployment/-/knowledge_base/7-1/installing-a-search-engine#choosing-a-search-engine)
```
Comment (lipusz) [inline discover/deployment/articles/01-installing-search-engine/03-installing-solr/00-installing-solr-intro.markdown]: I can't seem to find the "installing-a-search-engine" article on customer.liferay.com/documentation, while the LDN version says "Emporio": https://dev.liferay.com/en/discover/deployment/-/knowledge_base/7-1/installing-a-search-engine

Comment (rbohl) [reply discover/deployment/articles/01-installing-search-engine/03-installing-solr/00-installing-solr-intro.markdown]: fcb22c4c26a0da7a82eaeaa5fbcebab24b6e8f44

This commit removes the 00-intro file from `articles-dxp`. So the same intro will be used on customer.liferay.com and dev.liferay.com, once the customer portal's documentation sync process runs (after this branch is merged with master). So I think we're okay.

I used the @commerce@ tag, so it should properly name commerce on both the community site and the customer portal.

**Comment by lipusz** [comment]
Also, maybe we should do the reorg for 7.1 only. For 7.0, customers, people may have already bookmarked the pages and I don't known if we have a redirect mechanism in place

**Comment by rbohl** [comment]
@lipusz regarding the reorg., I think it 's most important once Installing Solr is published. It's true that the Configuring Elasticsearch article (and some of the other planned articles) are not just for LES customers, but Solr was the main reason we wanted to this reorg, so I think it's fine to wait. Although I recognize it was probably a mistake to combine the reorg. with the Solr docs. Ideally, I'd have done this much earlier. What do you think, can we afford to wait until Solr is published/

**Comment by lipusz** [comment]
If you can exclude the Solr guide from it you can publish the reorg right away.

**Comment by rbohl** [comment]
Yeah we have a `new-articles` folder I might be able to move them to temporarily. This folder isn't published, so it will be safe there until the Solr connector is ready for release. Then I'll move the articles back into the Installing a Search Engine section and we can publish them.

Sent the reorg here:
https://github.com/sez11a/liferay-docs/pull/3472

Do you consider the SME review of Installing Solr to be completed?

**Pull request merged**